.sync/workflows/leaf/codeql.yml

```
@@ -321,19 +321,41 @@ jobs:
       if: steps.codeqlcli_cache.outputs.cache-hit != 'true'
       run: stuart_update -c .pytool/CISettings.py -t DEBUG -a ${{ matrix.archs }} TOOL_CHAIN_TAG=${{ matrix.tool_chain_tag }} --codeql
 
+    - name: Find pytool Plugin Directory
+      id: find_pytool_dir
+      shell: python
+      run: |
+        import os
+        import sys
+        from pathlib import Path
+
+        # Find the plugin directory that contains the Compiler plugin
+        plugin_dir = list(Path(os.environ['GITHUB_WORKSPACE']).rglob('.pytool/Plugin/CompilerPlugin'))
+
+        # This should only be found once
+        if len(plugin_dir) == 1:
+            # If the directory is found get the parent Plugin directory
+            plugin_dir = str(plugin_dir[0].parent)
+
+            with open(os.environ['GITHUB_OUTPUT'], 'a') as fh:
+                print(f'codeql_plugin_dir={plugin_dir}', file=fh)

```

This will overwrite the valid CodeQL plugin location previously stored.

```suggestion
```

---

Looks like you're using the output later as `pytool_plugin_dir`. So:

```suggestion
            with open(os.environ['GITHUB_OUTPUT'], 'a') as fh:
                print(f'pytool_plugin_dir={plugin_dir}', file=fh)
```

---

You're correct.  I meant to change this to be `pytool_plugin_dir` which I'll fix now.